app/src/main/java/com/flxrs/dankchat/MainActivity.kt

```diff
@@ -191,6 +191,11 @@ class MainActivity : AppCompatActivity(), AddChannelDialogResultHandler,
         setSupportActionBar(binding.toolbar)
         updateViewPagerVisibility()
         fetchStreamInformation()
+        val oauth = preferenceStore.getOAuthKey()?.substringAfter("oauth:")
```

Comment:
Move this code block to line 98 to ensure ignores get loaded before recent messages